tests/operator_test.go

```diff
@@ -32,6 +32,8 @@ import (
 	"path/filepath"
 	"strconv"
 
+	"k8s.io/apimachinery/pkg/fields"
```

Comment:
it belong to line 51 with the rest of apimachinery please

Reply:
Done